README.md

```diff
@@ -1,36 +1,47 @@
 
 ## Overview
 
 Add analytics to your Java software with the **[Snowplow][snowplow]** event tracker for **[Java][java]**. See also: **[Snowplow Android Tracker][snowplow-android-tracker]**.
 
-With this tracker you can collect event data from your Java-based desktop and server apps, servlets and games. Supports JDK7+.
+With this tracker you can collect event data from your Java-based desktop and server apps, servlets and games. Supports JDK8+.
 
-## Quickstart
+## Find out more
+
+| Snowplow Docs                   | Contributing                      |
+|---------------------------------|-----------------------------------|
+| ![i1][techdocs-image]           |  ![i4][contributing-image]         |
+| **[Snowplow Docs][techdocs]**  | **[Contributing](CONTRIBUTING.md)**  |
+
+## Maintainer Quickstart
+
+Feedback and contributions are very welcome. If you have identified a bug, please log an issue on this repo. For all other feedback, discussion or questions please open a thread on our [Discourse forum][forums]. Feel free to make Pull Requests for new features, if you can code them yourself!
 
-Assuming git, **[Vagrant][vagrant-install]** and **[VirtualBox][virtualbox-install]** installed:
+Clone this repo and navigate into the cloned folder. To run the tests locally, you will need Docker or Java installed. Using either method, the build will fail if there are failing tests.  
+
+To run the tests using Docker, run:
 
 ```bash
- host$ git clone https://github.com/snowplow/snowplow-java-tracker.git
- host$ cd snowplow-java-tracker
- host$ vagrant up && vagrant ssh
-guest$ cd /vagrant
-guest$ ./gradlew clean build
-guest$ ./gradlew test
-guest$ ./gradlew publishToMavenLocal
-guest$ cd /examples/simple-console
-guest$ ./gradlew jar
-guest$ java -jar ./build/libs/simple-console-all-0.0.1.jar "http://<your-collector-domain>"
+$ docker build . -t snowplow-java-tracker
 ```
 
-## Find out more
+To run the tests using your installed JDK, run:
 
-| Technical Docs                  | Setup Guide               | Roadmap                 | Contributing                      |
-|---------------------------------|---------------------------|-------------------------|-----------------------------------|
-| ![i1][techdocs-image]           | ![i2][setup-image]        | ![i3][roadmap-image]    | ![i4][contributing-image]         |
-| **[Technical Docs][techdocs]**  | **[Setup Guide][setup]**  | **[Roadmap][roadmap]**  | **[Contributing](Contributing.md)**  |
+```bash
+$ ./gradlew build
+```
+
+We have also included a simple demo, found in the `examples/simple-console` folder. You will need a JDK installed to run it. When run, it sends several events to your event collector.
+
+```bash
+$ ./gradlew publishToMavenLocal
+$ cd examples/simple-console
+$ ./gradlew jar
+$ java -jar ./build/libs/simple-console-all-0.0.1.jar "http://<your-collector-domain>"
```

Comment:
This works now, because of the new `gradle-wrapper.jar` in simple-console.
I tried to add these steps in the Dockerfile but found a new problem - how to access/use MavenLocal with Docker. The build in `./gradlew jar` fails because "Could not resolve com.snowplowanalytics:snowplow-java-tracker:0.10.1."

Reply:
👍  np, we can give a look at that after the release in case